### ISSUE TITLE
Add test coverage for text displayed on Meetup link

### ISSUE THREAD

**Issue by schneidmaster** [clerb/clerb-rails#2]
In the events list, we display "View on Meetup" if the event is in the past or "RSVP on Meetup" if the event is in the future:

<img width="973" alt="screenshot 2018-09-18 12 46 50" src="https://user-images.githubusercontent.com/1896112/45702916-fc4ce900-bb40-11e8-8ee1-4064909a43f9.png">

It would be good to add test coverage for this distinction. I think it probably makes sense to put this under the [events feature spec](https://github.com/clerb/clerb-rails/blob/master/spec/features/events_feature_spec.rb). It may also be helpful to add the [timecop](https://github.com/travisjeffery/timecop) gem which lets you freeze time (so it's easy to set up "event is in the past" or "event is in the future" cases without the test being fragile as time passes).

**Comment by edmistond** [comment]
Hope you don't mind - I took a crack at this. :) 

I've been working with a ton of date-related manipulation in other languages recently so it was kind of interesting to see how this worked - Timecop is definitely a less painful way to do it than some of the other approaches I've tried.

Anyway, this is actually my first time working with rspec and Rails testing, so any feedback here is quite welcome. Thanks!
